extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -439,6 +445,9 @@ export class AmpA4A extends AMP.BaseElement {
       }
     }
 
+    this.detectedFont_ =
```

Comment:
Why do we need to put this in A4A as opposed to just calculating within googleAdUrl via a4a.win...?  Is the idea that you want to cache it?  If so, you could just write it on window as a variable.